internal/restic/parallel.go

```diff
@@ -69,8 +68,6 @@ func ParallelRemove[FT FileTypes](ctx context.Context, repo RemoverUnpacked[FT],
 		return nil
 	})
 
-	bar.SetMax(uint64(len(fileList)))
```

Comment:
Loosing the information about the total number of files that should be removed is a major problem in my opinion.

In fact, it's not necessary to remove the progress.Counter handling from ParallelRemove. The changed `report` functions are equivalent to the following change in `ParallelRemove`:
```go
err := repo.RemoveUnpacked(ctx, fileType, id)
if err != nil {
    bar.Add(1)
}
if report != nil {
    err = report(id, err)
}
```